package.json

```
@@ -5,12 +5,21 @@
   "homepage": ".",
   "license": "(EUPL-1.2)",
   "dependencies": {
+    "fuzzysort": "^1.1.4",
+    "js-sha256": "^0.9.0",
     "mdbreact": "mdbreact-4.26.1.tgz",
     "node-sass": "^4.12.0",
+    "obelisk.js": "^1.2.2",
     "react": "^16.8.6",
+    "react-chartjs-2": "^2.9.0",
     "react-dom": "^16.8.6",
+    "react-dropzone": "^11.0.1",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
+    "react-text-loop": "^2.3.0",
+    "react-typed": "^1.2.0",
+    "snek-intel": "file:snek-intel-1.0.0.tgz",
```

This should be changed when there is the first intel release.
```suggestion
    "snek-intel": "^0.1.0",
```